src/supported-value.js

```diff
@@ -35,11 +63,64 @@ export default function supportedValue(property, value) {
 
   // It is a string or a number as a string like '1'.
   // We want only prefixable values here.
-  if (typeof value !== 'string' || !isNaN(parseInt(value, 10))) return value
+  if ((typeof value !== 'string' && !Array.isArray(value)) ||
+    !isNaN(parseInt(value, 10))) {
+    return value
+  }
+
+  let isDoubleArray = false
+  let isArray = false
+
+  if (Array.isArray(value)) {
+    if (Array.isArray(value[0])) {
+      isDoubleArray = true
+      let doubleArrayValue = ''
+      for (let a = 0; a < value.length; a++) {
+        if (Array.isArray(value[a])) {
+          for (let v = 0; v < value[a].length; v++) {
+            if (v + 1 !== value[a].length) {
+              doubleArrayValue += `${value[a][v]} `
+            }
+            else {
+              doubleArrayValue += value[a][v]
+            }
+          }
+        }
+        else {
+          doubleArrayValue += value[a]
+        }
+        if (a + 1 !== value.length) {
+          doubleArrayValue += ', '
+        }
+      }
+      value = doubleArrayValue
+    }
+    else {
+      isArray = true
+      let arrayValue = ''
+      for (let v = 0; v < value.length; v++) {
+        if (v + 1 !== value.length) {
+          arrayValue += `${value[v]}, `
+        }
+        else {
+          arrayValue += value[v]
+        }
+      }
+      value = arrayValue
+    }
+  }
 
   const cacheKey = property + value
 
-  if (cache[cacheKey] != null) return cache[cacheKey]
+  if (cache[cacheKey] != null) {
+    if (isArray) {
+      return stringToArray(cache[cacheKey])
```

Comment:
It seems like we get an array, cache a string, then convert string back to array once we hit the cache, why? can't we cache the array directly?